### PR TITLE
Gaming Dashboard overhaul

### PR DESCRIPTION
**Pull request by RyRy79261**
Updated with fixes, this PR does not include design changes as functionality almost complete API side

### CODE REVIEW

File: packages/gaming-ui/src/Components/Modules/ApiKeys.tsx [hunk 1]
```diff
@@ -82,6 +84,57 @@ const useStyles = makeStyles(({ constants, breakpoints, animation }: CSGTheme) =
...
         fill: constants.fileSystemItemRow.menuIcon
       }
+    },
+    modalRoot: {
+      zIndex: zIndex?.blocker,
+      [breakpoints.down("md")]: {}
+    },
+    modalInner: {
+      // backgroundColor: constants.createFolder.backgroundColor,
+      // color: constants.createFolder.color,
```
Comment (Tbaut) [inline packages/gaming-ui/src/Components/Modules/ApiKeys.tsx]:
```suggestion
```

**Review by Tbaut — COMMENTED** [review]
I'm surprised to see the api keys and s3 key from storage, is that expected?

image

Pushing a commit to replace the files logo/favicon

File: packages/gaming-ui/src/Components/Pages/LoginPage.tsx [hunk 2]
```diff
@@ -138,10 +138,11 @@ const LoginPage = () => {
             <BottomDarkSVG className={classes.bgBottom} />
             <TopDarkSVG className={classes.bgTop} />
           </>
-          : <>
+          :          <>
```
Comment (tanmoyAtb) [inline packages/gaming-ui/src/Components/Pages/LoginPage.tsx]:
```suggestion
          : <>
```

File: packages/gaming-ui/src/Components/Pages/LoginPage.tsx [hunk 3]
```diff
@@ -138,10 +138,11 @@ const LoginPage = () => {
...
             <BottomLightSVG className={classes.bgBottom} />
             <TopLightSVG className={classes.bgTop} />
           </>
+
```
Comment (tanmoyAtb) [inline packages/gaming-ui/src/Components/Pages/LoginPage.tsx:145]:
```suggestion
```

**Review by tanmoyAtb — APPROVED** [review]
A couple of nits, otherwise what Tbaut asked about the keys, not very sure about the functionality!

**Comment by FSM1** [comment]
> I'm surprised to see the api keys and s3 key from storage, is that expected?
> 
> image
> 
> Pushing a commit to replace the files logo/favicon

My guess is that this is just to bring the current repo in line with what was done on Storage - The actual functionality of requesting API keys still needs to be fleshed out.

**Comment by RyRy79261** [comment]
@FSM1 @Tbaut So basically key generation is ready but we're finishing up the kong management section, though why you'd be seeing keys already I'm not sure

@sadiq1971 Might have some insight

**Review by Tbaut — APPROVED** [review]
ok I'd say there's no need to delay this PR more, let's merge and keep this in mind.

**Pull request merged**